src/enum.cr

```diff
@@ -307,6 +307,20 @@ struct Enum
     {% end %}
   end
 
+  # Returns the number of enum members for normal enum, or the number of flags for flags enum.
+  #
+  # ```
+  # Color.size  # => 3
+  # IOMode.size # => 3
+  # ```
+  def self.size : Int32
+    {% if @type.has_attribute?("Flags") %}
+      {{ @type }}::All.value.popcount
```

Comment:
Do we still want to dedupe for flags, and not for non-flags enums?

Reply:
It seemed to be preferable to have `Enum.size` return the number of members for non-flags, I guess I should change it for flags too?

Or even rename it to `Enum.count_members` to avoid any mis-interpretation?